src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -74,30 +75,48 @@ def __init__(
             for i in range(len(shares)):
                 self.shares.append(self._encode(shares[i]).to(tensor_type))
 
-    def _encode(self, data):
+    def _encode(self, data: torch.Tensor) -> torch.Tensor:
+        """Encode via FixedPointEncoder.
+
+        Args:
+            data (torch.Tensor): Tensor to be encoded
+
+        Returns:
+            encoded_data (torch.Tensor): Decoded values
```

Comment:
I think here it should be `Encoded vales` in the docstring.

Reply:
Its more or less the same.